gr-qtgui/lib/fft_shift.h

```diff
@@ -0,0 +1,67 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2012,2014-2015 Free Software Foundation, Inc.
```

Comment:
2019

Reply:
I have added "2019" also to changed files in gr-qtgui. Is this ok?